proto/panacea/oracle/v2alpha2/genesis.proto

```diff
@@ -1,24 +1,17 @@
 syntax = "proto3";
 package panacea.oracle.v2alpha2;
 
-option go_package = "github.com/medibloc/panacea-core/x/oracle/types";
+option go_package = "github.com/medibloc/panacea-core/v2/x/oracle/types";
 
 import "gogoproto/gogo.proto";
 import "google/protobuf/duration.proto";
 import "google/protobuf/timestamp.proto";
+import "panacea/oracle/v2alpha2/oracle.proto";
 
 // GenesisState defines the oracle module's genesis state.
 message GenesisState {
-}
-
-// TallyResult defines the tally result of the votes.
-message TallyResult {
```

Comment:
I moved this to `oracle.proto`, a cycle error occurred, so I took action.

f it is good that the message exists here, I will return it and find out if there is another way.